tlslite/keyexchange.py

```diff
@@ -229,40 +229,41 @@ def makeClientKeyExchange(self):
         clientKeyExchange.createRSA(self.encPremasterSecret)
         return clientKeyExchange
 
-# the DHE_RSA part comes from IETF ciphersuite names, we want to keep it
-#pylint: disable = invalid-name
-class DHE_RSAKeyExchange(KeyExchange):
-    """
-    Handling of ephemeral Diffe-Hellman Key exchange
 
+class ADHKeyExchange(KeyExchange):
+    """
+    Handling of anonymous Diffe-Hellman Key exchange
+    FFDHE without signing serverKeyExchange useful for anonymous DH
     NOT stable API, do NOT use
     """
 
     def __init__(self, cipherSuite, clientHello, serverHello, privateKey):
```

Comment:
And theres no problem, when I change 
`def __init__(self, cipherSuite, clientHello, serverHello, privateKey):` to 
`def __init__(self, cipherSuite, clientHello, serverHello, privateKey=None):` ?